src/web/forms/fields/AutoCalc/__tests__/functions-test.js

```diff
@@ -42,3 +42,57 @@ describe('asha:bmi', () => {
       .toBeCloseTo(21.46);
   });
 });
+
```

Comment:
Maybe also it would be good to have test case for mixed int (0-precision) number & float (1+ decimal point precision) number?

Reply:
Good suggestion.
I thought it was not necessary because, in JS, all numbers are float and there is no int number.
But more test cases there are, it is better.
I will add it.

Reply:
I created new PR: #686